ledger/voters.go

```diff
@@ -61,8 +62,8 @@ type votersTracker struct {
 	// for round X+CompactCertVotersLookback+CompactCertRounds.
 	round map[basics.Round]*ledgercore.VotersForRound
 
-	l  ledgerForTracker
-	au *accountUpdates
+	l   ledgerForTracker
+	hdr bookkeeping.BlockHeader
```

Comment:
It would be good to document what this hdr is (what round number is this expected to correspond to)?

From the code, it looks like this is the header of the most recent block seen by this tracker (so, roughly the latest block in the ledger, modulo some transient effects).  It's a bit odd why this needs to be stored here, though.  It seems like you only use this to decide if compact certs are enabled.  If that's the only use, maybe it makes more sense to store a flag indicating whether compact certs are enabled?  Or maybe fetch the header in committedUpTo(), for the committed round, so no additional state would be needed at all?